src/idp/response_builder.rs

```diff
@@ -49,19 +50,19 @@ fn build_attributes(formats_names_values: &[ResponseAttribute]) -> Vec<Attribute
             friendly_name: None,
             name: Some(attr.required_attribute.name.clone()),
             name_format: attr.required_attribute.format.clone(),
-            values: attr.values.iter().map(|value| {
-                AttributeValue {
+            values: attr
+                .values
+                .iter()
+                .map(|value| AttributeValue {
                     attribute_type: Some("xs:string".to_string()),
                     value: Some(value.to_string()),
```

Comment:
Aaah. This is silly. Why would they require `&'a str` in the first place, if they are just going to convert to String immediately after.

now I see what you were talking about. If `AttributeValue` also had lifetimes and `&'a str`, and it was converted to JSON before the lifetimes would expire, then sure I understand why they would do it this way.


`Vec<String>` is the way to go, forget about lifetimes.

Reply:
Fixed with fa010883f08068d2db54f1abc099d7a773055437.

Reply:
Went with `Vec<String>` for the moment but we can discuss how we should ultimately solve this at https://github.com/spd-mfa/samael/commit/c3fc9f1b093bd7991aeb9dbc84e2c79eb64d208a#r112786776. Sounds like we'll want Cow + lifetimes after all.